### ISSUE TITLE
Increasing BA*+D215N 

### ISSUE THREAD

**Issue by RajLABN** [cov-lineages/pango-designation#442]
@Gabriel_H_R/@Seba1337 alerted about this omicron mutation in twitter on 2/13/22 and probably some others too

Curious about increasing omicron sequences with D215N in Croatia (3723), Estonia (484), Latvia (660)...

Looked at the sequences uploaded and assigned mutations in GISAID
Most of these sequences also had Spike_ins212RG instead of the typical spike_ins214EPE

9 sequences from USA (in the same group) and 23 over all had Spike_ins215PEN

While I am looking at this, I want others to have a look at this as well...

[https://cov-spectrum.org/explore/World/AllSamples/AllTimes/variants/international-comparison?aaMutations=s%3AD215N&pangoLineage=B.1.1.529*]

**Comment by corneliusroemer** [comment]
I already commented on this on Twitter, I don't think this is real, it's likely a sequencing artefact because the nearby insertion is somehow misassembled.

You can't just go by the number of sequences containing a mutation. If you think this is real, please show that this isn't just an artefact by having a look at sequences displaying this putative mutation.

I think @thomasppeacock agreed with me on the nature of this artefact.

So I'll close this issue unless there's evidence to the contrary.

Twitter comments:
https://twitter.com/CorneliusRoemer/status/1493155617711742979?s=20&t=cFGUJpHUjFtPPfGodOikcA
https://twitter.com/CorneliusRoemer/status/1493155186814210049?s=20&t=cFGUJpHUjFtPPfGodOikcA

**Comment by RajLABN** [comment]
Thank you @corneliusroemer

**Comment by seba1337** [comment]
This one isn't the one I've originally found, but GISAID decided on totally different AAs.
https://twitter.com/seba1337/status/1493601899722985476

The original one, which I've found, is most likely a frameshift + insert, but who knows what happens with D215 when it's translated in practice. I doubt it comes out as D.
https://twitter.com/seba1337/status/1493563884170616833

**Comment by RajLABN** [comment]
>>update <<  @seba1337 @corneliusroemer 

GISAID has fixed some of the unusual inserts - 

" The Spike_ins212RG and Spike_ins215PEN that you've seen earlier were called by blastx protein alignment. Although the protein alignment produced by blastx was parsimonious and made sense, it did not reflect the actual mutational events that is observed at the nucleotide level. We have since corrected these variants to match the results from codon-based nucleotide alignments. These insertions (Spike_ins212RG and Spike_ins215PEN) are no longer present in the EpiCoV database and has been corrected to become Spike_ins214GAR and Spike_ins214EPE + Spike_D215N respectively."

Without access to their fastq files, **GISAID curators are unable to determine if the "ggagccaga (GAR)" insertion is due to a sequence assembly artefact, but because there has been over 20 different submitting labs producing the same sequence**, we are currently accepting these insertions as they are being represented in the final consensus sequence.

GISAID has contacted some of these submitters to submit their fastq files for these sequences to confirm this insertion with the read data.
------------------------------------
As of now, 7541 sequences with spike_214insGAR (7526 Omicron, 8 Delta and 1 B.1.1 with sample collection dates in Nov 2021 to Feb 2022. 

7525 of these sequences are from Europe and 5420 of these sequences are from Croatia.

Hope some of these submitters send their fastq files to GISAID curators!
